src/axislike.rs

```diff
@@ -688,3 +703,22 @@ impl From<DualAxisData> for Vec2 {
         data.xy
     }
 }
+
+/// The shape of the deadzone for a [`DualAxis`] input.
+/// 
+/// Uses the x and y thresholds on [`DualAxis`] to set the shape diamensions
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub enum DeadzoneShape{
+    /// Deadzone with a cross
+    /// 
+    /// Uses the x threshold as the horizontal width and y threshold as the vertical width.
+    Cross,
+    /// Deadzone with a rectangle.
+    /// 
+    /// Uses the x threshold as the width and y threshold as the length.
+    Rect,
+    /// Deadzone with a circle.
+    /// 
+    /// Uses the larger of the x or y threshold as the radius
```

Comment:
I think we should also have an ellipse. I could see an argument either way for replacing the circle completely, so I'll leave it up to you.

Reply:
I agree with removing the circle and adding an ellipse. It provides more control over a circle. It is the same reason I went for using a rectangle over a square.